tests/scripts/helm.sh

```diff
@@ -26,19 +24,31 @@ detectArch() {
     esac
 }
 
-install() {
-    if ! helm_loc="$(type -p "helm")" || [[ -z ${helm_loc} ]]; then
-        # Download and unpack helm
-        local dist
-        dist="$(uname -s)"
+# Echo the name or path of the helm command. Install helm to a temp location
+# if it's not present.
+install_helm() {
+    local helm="helm"
+    local dist="$(uname -s)"
+    dist="$(echo "${dist}" | tr '[A-Z]' '[a-z]')"
+    local tmp_helm="${temp}/${dist}-${arch}/helm"
+
+    if [[ -f "${tmp_helm}" ]]; then # helm installed by this script
+        helm="${tmp_helm}"
+    elif ! which ${helm} &>/dev/null; then
+        echo "Installing helm..." >&2
         # shellcheck disable=SC2021
```

Comment:
You probably need to remove the `shellcheck` comment now unless it is needed. Could you check that with shellcheck if this is still needed?

Reply:
Thanks for your reviews. I've removed this.